.github/workflows/newcomers-alert.yml

```diff
@@ -0,0 +1,16 @@
+name: Newcomers Alert
```

Comment:
@lolverae, oh, now I see what you mean about a missing slack workflow. On today's call, I thought you were referring to the Slack workflow that will send a message when someone stars a repository. So, thanks for catching this one. 👍